internal/elasticsearch/ingest/datastreamstats.go

```diff
@@ -0,0 +1,54 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package ingest
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/elasticsearch"
+)
+
+type DataStreamsStats struct {
+	DataStreams []DataStreamStats `json:"data_streams"`
+}
+
+type DataStreamStats struct {
+	DataStream       string `json:"data_stream"`
+	BackingIndices   int    `json:"backing_indices"`
+	StoreSizeBytes   int    `json:"store_size_bytes"`
+	MaximumTimestamp int    `json:"maximum_timestamp"`
+}
+
+func GetDataStreamStats(esClient *elasticsearch.API, datastream string) (*DataStreamStats, error) {
+	req := esClient.Indices.DataStreamsStats.WithName(datastream)
+	resp, err := esClient.Indices.DataStreamsStats(req)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Node Stats API call failed")
```

Comment:
nit. DataStream Stats API? 
```suggestion
		return nil, fmt.Errorf("failed call to DataStream Stats API: %w", err)
```